File: app/src/main/java/Controller/JsonplaceHolderApi.java
package Controller;

import java.util.ArrayList;
import java.util.Map;

import models.Movie;
import models.Result;
import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface JsonplaceHolderApi {
    @GET("popular")
    Call<Result> get_Result(@Query("api_key") String key,@Query("page")int page);

}

File: app/src/main/java/com/example/apimovie/Api/JsonplaceHolderApi.kt
package com.example.apimovie.Api

import com.example.apimovie.models.ResultMovie
import kotlinx.coroutines.Deferred
import retrofit2.http.GET
import retrofit2.http.Query


interface JsonplaceHolderApi {

    @GET("popular")
    fun get_Result(@Query("api_key") key: String?, @Query("page") page: Int): Deferred<ResultMovie>?
}